ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/PrivacyParameters.java

```diff
@@ -132,6 +133,14 @@ public void setEnclave(final Enclave enclave) {
     this.enclave = enclave;
   }
 
+  private void setMultiTenancyEnabled(final boolean multiTenancyEnabled) {
+    this.multiTenancyEnabled = multiTenancyEnabled;
+  }
+
+  public boolean isMultiTenancyEnabled() {
+    return multiTenancyEnabled;
+  }
+
   @Override
   public String toString() {
     return "PrivacyParameters{" + "enabled=" + enabled + ", enclaveUri='" + enclaveUri + '\'' + '}';
```

Comment:
may be add multi-tenancy in toString method as well?

Reply:
done